rs/execution_environment/tests/subnet_size_test.rs

```diff
@@ -93,7 +93,9 @@ fn inc_instruction_cost(config: HypervisorConfig) -> u64 {
         0
     };
 
-    5 * cc + cs + cl + ca + 2 * ccall + csys + cd
+    let func_call_cost = 1; // Calling a function costs 1 instruction.
```

Comment:
instead of assigning 1, just to make it future proof perhaps call `instruction_to_cost(&wasmparser::Operator::Func, ...)`

Reply:
There's no such operator as `wasmparser::Operator::Func`. I also tried to find anything close to `Call` or `RefCall`, but those have cost different from 1. also neither `Return` nor `Drop` (=1) come here, so I renamed that constant of 1 to a default cost of unknown operator. WDYT?